src/compressed_tensors/quantization/quant_args.py

```diff
@@ -257,18 +279,31 @@ def validate_model_after(model: "QuantizationArgs") -> "QuantizationArgs":
             if strategy not in (
                 QuantizationStrategy.TOKEN,
                 QuantizationStrategy.TENSOR,
+                QuantizationStrategy.TENSOR_GROUP,
             ):
                 raise ValueError(
-                    f"One of {QuantizationStrategy.TOKEN} or "
-                    f"{QuantizationStrategy.TENSOR} must be used for dynamic ",
-                    "quantization",
+                    f"One of {(QuantizationStrategy.TOKEN, QuantizationStrategy.TENSOR, QuantizationStrategy.TENSOR_GROUP)} "
+                    "must be used for dynamic quantization",
                 )
+
+            if (
+                dynamic == DynamicType.LOCAL
+                and strategy != QuantizationStrategy.TENSOR_GROUP
```

Comment:
I missed this in our slack discussion. If this is the case, might it be better to expand the enum with something like `QuantizationStrategy.TENSOR_GROUP_LOCAL` and `QuantizationStrategy.TENSOR_GROUP_GLOBAL` to avoid having `Union[DynamicType, bool]` types in the API? I may very well be missing something here though

Reply:
I dont think we want to combine strategies and dynamic types. QuantizationStrategy.TENSOR_GROUP_LOCAL and QuantizationStrategy.TENSOR_GROUP_GLOBAL wont work as we want to provide a distinction between the dynamic generation and the strategy. You can have global and local scales which are purely dynamic

Reply:
ok, I suppose we'll have presets for these anyway, so users won't have to worry about hitting these errors